app/src/main/res/values/dimens.xml

```diff
@@ -587,5 +587,6 @@
   <dimen name="section_title_text_view_layout_margin_start">28dp</dimen>
   <dimen name="section_title_text_view_layout_margin_top">28dp</dimen>
   <dimen name="section_title_text_view_layout_margin_end">32dp</dimen>
-
+  <!-- StoryFragment-->
+  <dimen name="story_fragment_padding_bottom">160dp</dimen>
```

Comment:
Add a space above the `<!-- StoryFragment-->` comment. Also remove the `bottom_white_space` dimension from this file as it is no more needed now.

Reply:
Okk,I will apply these changes

Reply:
@ARJUPTA On removing bottom_white_space it is showing error.Maybe it is used in any other xml.